tests/casp/journal_check.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic CASP journal tests
+# Maintainer: Tomas Hehejik <thehejik@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub write_detail_output {
```

Comment:
I see. good idea. It is also a good idea to try it out first in special casp test, then move it "upwards", e.g. first into something like `lib/utils.pm` as part of the tests distribution and then further to the test api of os-autoinst. I think this is what you intended :-)